### ISSUE TITLE
Image after capture not obtained in Android 11

### ISSUE THREAD

**Issue by ghost** [mayuce/AndroidDocumentScanner#41]

**Comment by mayuce** [comment]
Could you add more description please? Is it only some images or all the images are not obtained? Did you compare it with any other API versions?

**Comment by nedushny** [comment]
@mayuce hi, I used API 32 (Android 12) but after capturing an image through the camera, no image is output.
API 26 works for me, but there is a problem with image rotation (image rotate to 90 degree). Picture below

![image](https://user-images.githubusercontent.com/71440998/173629760-5b5d468c-d98e-4178-a192-b25e62204f51.png)

**Comment by KamranKhanDemo** [comment]
Yes Image rotation is the problem, and I know the class it is condition based NativeClass.getPoint
